src/elements/content-sidebar/messages.js

```diff
@@ -39,7 +39,7 @@ const messages = defineMessages({
     },
     boxSignFtuxTitle: {
         id: 'be.contentSidebar.boxSignFtuxTitle',
-        defaultMessage: 'Box Sign - Secure, seamless e-signatures in Box',
+        defaultMessage: 'Box Sign - Secure, seamless e‑signatures in Box',
```

Comment:
is this change intentional? while the dash characters appear the same, I don't see usages of this particular ASCII character in BUIE or some of the other frontend repos

Reply:
The change is intentional and replaces the standard breaking hyphen character with its non breaking variant (U+2011) since "e-signatures" it's a single word and should not be broken by browsers like in the image below:

<img width="497" alt="Screenshot 2023-11-28 at 18 42 33" src="https://github.com/box/box-ui-elements/assets/152302653/a070a697-179f-4d58-be10-3818d5762b33">